Automattic/jetpack
# Remove Pocket Casts embed block variation

**Changes proposed in this Pull Request:**
This PR removes the Pocket Casts embed block variation, it was added on https://github.com/Automattic/jetpack/pull/23740 for internal dev purposes.

The Pocket Casts block will be moved to Gutenberg on https://github.com/WordPress/gutenberg/pull/41006.

**Jetpack product discussion:**
pdDOJh-bf-p2 and pdeCcb-dg-p2

**Does this pull request change what data or activity we track or use?**
No

**Testing instructions:**
1. Open the block editor
2. Search for the Pocket Casts block
3. The block should not appear in the results
![image](https://user-images.githubusercontent.com/3113712/169623176-1606a0ef-a64a-4077-a75d-44faf9992b3e.png)
